test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -1975,6 +1980,10 @@ public Settings transportClientSettings() {
         };
     }
 
+    protected List<Settings> updateNodesSettings(List<Settings> allNodesSettings) {
```

Comment:
I think this name is too vague. How about `addExtraClusterBootstrapSettings`? Also I think JavaDoc would be useful, to describe that this is called when starting a collection of nodes, and the settings that are passed in have the node names configured.

Reply:
The name of this method is general on purpose. I don't want to limit this method just for bootstrapping use case. Probably in the future, we will have more settings which could be set only once all nodes settings are available. I've added JavaDoc in this commit https://github.com/elastic/elasticsearch/pull/36559/commits/d69997fbf0e7b61898e15ca438d06f62fd1ea8ec

Reply:
I think we can rename it in future if it turns out that the extra generality is needed. Today, being more specific with the name will help clarify the meaning of this method in the places where it is used.

Reply:
Ok, I've renamed the method